docs/extensions/guides/stores.md

```diff
@@ -1,11 +1,154 @@
----
-WIP
----
-
 # Stores
 
-## ClusterStore
+Stores are components that persist and synchronize state data. Lens utilizes a number of stores for maintaining a variety of state information.
+A few of these are exposed by the extensions api for use by the extension developer.
+The `ClusterStore` manages cluster state data such as cluster details, and which cluster is active.
+The `WorkspaceStore` similarly manages workspace state data, such as workspace name, and which clusters belong to a given workspace.
+The `ExtensionStore` is a store for managing custom extension state data.
+
+## ExtensionStore
+
+Extension developers can create their own store for managing state data by extending the `ExtensionStore` class.
+This guide shows how to create a store for the [`appPreferences` guide example](../renderer-extension#apppreferences), which demonstrates how to add a custom preference to the Preferences page.
+The preference is a simple boolean that indicates whether something is enabled or not.
+The problem with this example is that the enabled state is not stored anywhere, and reverts to the default the next time Lens is started.
```

Comment:
```suggestion
The problem with that example is that the enabled state is not stored anywhere, and reverts to the default the next time Lens is started.
```

Reply:
Didn't you already approve this? :)

Reply:
Yes, just noticed these this time around